pydantic_ai_slim/pydantic_ai/mcp.py

```diff
@@ -507,9 +518,41 @@ async def client_streams(
             MemoryObjectSendStream[SessionMessage],
         ]
     ]:
+        r_fd, w_fd = os.pipe()
+        err_read_stream_writer, err_read_stream = anyio.create_memory_object_stream[str]()
```

Comment:
Do we still need this, or could `write_stderr` directly add onto `stderr_chunks` instead of going through `err_read_stream_writer` and `err_read_stream`? If so, we can rename it to `read_stderr`